lego/apps/email/serializers.py

```diff
@@ -12,6 +12,23 @@
 from .fields import EmailAddressField
 
 
+class PublicEmailListSerializer(serializers.ModelSerializer):
+
+    email = EmailAddressField(read_only=True)
+
+    class Meta:
+        model = EmailList
+        fields = (
+            "id",
+            "users",
+            "name",
+            "email",
+            "groups",
+            "group_roles",
+            "require_internal_address",
```

Comment:
Do you need all these fields? I have a feeling the `/me` response is getting quite large now :upside_down_face:.

Reply:
Nah, not that important i guess. But they are quite small however, so waaay smaller than memberships and groups...

Reply:
Also, this drops `/users/me` from 20 to 15 queries in dev, so it is probably faster now.

Reply:
Can probably move stuff out of `/me` to separate endpoints, but perms are pretty nice when using it, and we still "need" all the info when looking at the user profile on the webapp.

Reply:
Also, this only return ids, and not complete user and group entities, so fairly small.

Reply:
Hmm, yeah we prob have larger endpoints, so I guess it's fine :+1: